src/components/Question/ValidationAnswerTypes/ValidationAnswerTypeAttachment.tsx

```diff
@@ -21,14 +21,19 @@ const ValidationAnswerTypeAttachment = ({ item }: ValidationAnswerTypeAttachment
     }
 
     return (
-        <FormField label={t('Max file size in MB')}>
+        <>
+        <FormField 
+            label={t('Max file size in MB')}
+            sublabel={t('Note: File size validation is not currently supported by ResearchKitonFHIR.')}
```

Comment:
Similar comment as noted above:
```suggestion
            sublabel={t('Note: File size validation is not currently supported by <a href="https://github.com/StanfordBDHG/ResearchKitOnFHIR">ResearchKitOnFHIR</a>.')}
```

Reply:
Can't render a link in this label, but the link has been rendered in the previous code, so that should be sufficient.